src/content-helper/common/components/persona-selector/component.tsx

```diff
@@ -11,63 +11,57 @@ import {
 import { useDebounce } from '@wordpress/compose';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import { Icon, chevronRight } from '@wordpress/icons';
+import { Icon, edit } from '@wordpress/icons';
+import { MoreArrow } from '../../icons/more-arrow';
```

Comment:
Let's move this line under internal dependencies.

Reply:
Oops! Good catch.